setup.py

```diff
@@ -155,9 +155,14 @@
 ]
 extras_require = {
     ':sys_platform != "win32"': ['terminado>=0.3.3'],
-    'doc': ['Sphinx>=1.1'],
+    'doc': [
+        'Sphinx>=1.1',
```

Comment:
We pip install 1.3.6 within our conda environment.yml for building the docs on RTD. This version is needed for markdown sources if one wishes to have the "Edit on GitHub" button in the right hand of the RTD theme direct to the correct place.